rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector.php

```diff
@@ -36,35 +46,91 @@ public function __construct(
     public function getRuleDefinition(): RuleDefinition
     {
         return new RuleDefinition('Change return type based on strict scalar returns - string, int, float or bool', [
-            new CodeSample(
+            new ConfiguredCodeSample(
```

Comment:
This had to be changed because of implementing `ConfigurableRectorInterface`. Personally however I would find it confusing to have this rule described as configurable, including the `hard_coded_only` option, while there is a default so it doesn't need to be configured. But not sure if there is any method for that. (Using `[]` as `$configuration` parameter didn't work either though).